src/components/menus/TopMenu.js

```diff
@@ -72,7 +52,7 @@ export const TopMenu = ({ menu, activeDocMeta }) => {
               <div className={sty.SocialGroup}>
                 <ul className={`list-no-style ${sty.SocialLinks}`}>
                   {menu.socials.map((item, index) => (
-                    <li>
+                    <li key={index}>
```

Comment:
Would there be a better key than index to use or was it used for a specific reason?

Reply:
Nope great catch, I should pick a better key haha.